### PR TITLE
Add .mailmap to consolidate authorship (git shortlog -e)

### PR DESCRIPTION
**Pull request by dolmen**
Cc: @webron @fehguy @xhh @mohsen1 @silas @wting

### CODE REVIEW

**Comment by wting** [comment]
Looks good to me, my emails are mapped correctly.

**Comment by RobDolinMS** [comment]
I'm not familiar with mailmap, but I did a bit of digging and found documentation at: https://www.git-scm.com/docs/git-check-mailmap

**Comment by RobDolinMS** [comment]
@dolmen @wting - It looks like this consolidates on people's personal email addresses rather than their work email addresses.  Is this intentional?  (Am I possibly misinterpreting the file?)

**Comment by webron** [comment]
It's based on the commits. I don't really see a need for this.

**Comment by RobDolinMS** [comment]
@namdeirf Do we have a Bitergia (sp?) account for OAI?

If so, does it use the .mailmap file or does it use something else?

**Comment by RobDolinMS** [comment]
@webron I would be inclined to close this for now; and then re-open IF there is some additional need (such as my [comment to JeffEF](https://github.com/OAI/OpenAPI-Specification/pull/478#issuecomment-282844610)

File: .mailmap [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Mohsen Azimi <me@azimi.me> Mohsen Azimi <mazimi@apigee.com>
+Mohsen Azimi <me@azimi.me> Mohsen Azimi <msn.azi@gmail.com>
```
Comment (mohsen1) [inline .mailmap]: Please remove my name and email. I get enough emails for Swagger stuff already! 😁

**Comment by webron** [comment]
Closing for now. Thanks for the work though.

**Pull request closed without merging**